### PR TITLE
Clarify what happens with git remote rm command

### PR DESCRIPTION
**Pull request by peterjv26**
Added a note to say the "git remote rm" command  just unlinks local and remote repositories

### Why:

Closes [issue link]

### What's being changed:
Updated the doc adding a line to clarify what exactly happens with "git remove rm" command

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@peterjv26 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/get-started/getting-started-with-git/managing-remote-repositories.md [hunk 1]
```diff
@@ -194,6 +194,8 @@ Use the `git remote rm` command to remove a remote URL from your repository.
 The `git remote rm` command takes one argument:
 * A remote name, for example, `destination`
 
+Note: Removing the remote URL from your respositoy only unlinks the local and remote repositories and does not delete the remote repository
```
Comment (oNaiPs) [inline content/get-started/getting-started-with-git/managing-remote-repositories.md:197]: typo respositoy

**Comment by Soundpisey** [comment]
sorry,

File: content/get-started/getting-started-with-git/managing-remote-repositories.md (hunk 1 above)
Comment (skedwards88) [inline content/get-started/getting-started-with-git/managing-remote-repositories.md:197]:
```suggestion
Removing the remote URL from your repository only unlinks the local and remote repositories. It does not delete the remote repository.
```

**Review by skedwards88 — COMMENTED** [review]
Thank you for this addition! I made some minor wording changes that I will incorporate, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Review by Jase8338 — COMMENTED** [review]
$ git remote rm sofake
> error: Could not remove config section 'remote.sofake'

**Pull request merged**